source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -683,6 +683,20 @@ def __init__(self):
             help="used to adjust the search range in Asgard when computing matrix",
         )
 
+        parser_get.add_argument(
+            "_pt_journey_fare",
+            type=OptionValue(['kraken']),
+            hidden=True,
+            help="choose which pt engine to compute the pt journey's fare",
+        )
+
+        parser_get.add_argument(
+            "_compute_pt_journey_fare",
+            type=BooleanType(),
+            hidden=True,
+            help="whether use external engine to compute pt journey fare",
```

Comment:
Is this argument currently used to "force" (re-)computing of fares (even if using `kraken` for journey computing)? 
I'm not sure about what confuses me, but if it's also clearer to you, you may change to: 
```suggestion
            "_must_compute_pt_journey_fare_externally",
            type=BooleanType(),
            hidden=True,
            help="whether to use external engine to compute pt journey fare",
```